{
  "contest": 258,
  "handle": "seerether",
  "risk": "3",
  "title": "Claim failure can result to loss of funds",
  "issueId": 390,
  "issueUrl": "https://github.com/code-423n4/2023-07-pooltogether-findings/issues/390"
}